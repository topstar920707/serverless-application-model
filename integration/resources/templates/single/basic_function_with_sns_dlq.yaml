Resources:
  MyFunction:
    Type: AWS::Serverless::Function
    Properties:
      Handler: index.handler
      Runtime: nodejs12.x
      CodeUri: ${codeuri}
      DeadLetterQueue:
        Type: SNS
        TargetArn:
          Ref: "MyTopic"

  MyTopic:
    Type: AWS::SNS::Topic
