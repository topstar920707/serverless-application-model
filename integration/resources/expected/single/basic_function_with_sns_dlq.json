[
  { "LogicalResourceId":"MyFunction", "ResourceType":"AWS::Lambda::Function"},
  { "LogicalResourceId":"MyFunctionRole", "ResourceType":"AWS::IAM::Role" },
  { "LogicalResourceId":"MyTopic", "ResourceType":"AWS::SNS::Topic" }
]